Automattic/wp-calypso
# Marketplace: Edit marketplace CTA on /marketing/tools

#### Proposed Changes

Change the title colou from light grey to #101517 (--studio-gray-100)
Replace this illustration with the one on [Figma](f84fqtCG4pCnCh2Hq0But5-fi-1601%3A13181)

#### Testing Instructions

1. Go to `/marketing/tools`
2. Check the illustration and font color of the first block to match the screenshot. (The button depends on the theme selected)

#### Screenshot
<img width="915" alt="image" src="https://user-images.githubusercontent.com/97929097/172738387-41a58edc-adf0-49bc-96c4-21058397bc58.png">

Fixes #64474
